jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -65,8 +65,11 @@ Example usage:
        (let* (avs (minjerk (instance minjerk-interpolator :init))
                   (scale-av (send self :sub-angle-vector av prev-av))
                   dist div)
+         (setq tm (float tm))
          (setq dist (abs (geo::find-extream (coerce diff-av cons) #'abs #'>=)))
-         (setq div (round (/ dist 120.0)))
+         ;; Use small interpolation time
+         ;; https://github.com/euslisp/jskeus/pull/596#issuecomment-765213357
+         (setq div (* 10 (round (/ dist 120.0))))
```

Comment:
@708yamaguchi are you sure about this ? Is there any side effect of this change? for example, increasing processing time?

Reply:
Thank you for your review.

After https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/456 is merged, smaller `dt` of minjerk interpolation is required.
So I increase `div` and make `dt` smaller.
(With the current jsk_robot origin/master and jsk_pr2eus origin/master, minjerk-interpolation in :angle-vector-raw in fetch-interface.l does not work)

> Is there any side effect of this change?

The original purpose of `div` is to prevent sending over-180 degree angle-vector.
So I think increasing `div` does not change the robot motion.
https://github.com/jsk-ros-pkg/jsk_robot/pull/849

> for example, increasing processing time?

I compared the processing time of minjerk interpolation in :angle-vector-raw in fetch-interface.l.
The processing time increases more than 20x times, but I think it's not problem because 0.020229[s] is small enough.

1. Use jsk_robot origin/master and jsk_pr2eus 0.3.14, which does not include https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/456

    The processing time is `;; time -> 0.020029[s]` and `div` is 420.

2. Use jsk_robot of this PR and jsk_pr2eus origin/master

    The processing time is `;; time -> 0.000908[s]` and `div` is 42.

The test code is below.
I add `(bench)` and `(print div)` in `:angle-vector-raw`.
https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/fetcheus/fetch-interface.l#L69-L76

```
roseus fetch-interface.l
(fetch-init)
(send *ri* :angle-vector-raw (send *fetch* :init-pose) 5000)
;; print processing time and div
(send *ri* :angle-vector-raw (send *fetch* :angle-vector #f(5000.0 5000.0 5000.0 5000.0 5000.0 5000.0 5000.0 5000.0 5000.0 5000.0)) 5000)
```

original interpolation
```
(bench
         (setq div (round (/ dist 120.0)))
         (print div)
         (send minjerk :reset
               :position-list (list prev-av (v+ prev-av scale-av))
               :time-list (list tm))
         (send minjerk :start-interpolation)
         (send minjerk :pass-time (/ tm div))
         (dotimes (i div)
           (setq avs (append avs (list (send minjerk :pass-time (/ tm div))))))
)
```

smaller dt interpolation
```
(bench
         (setq div (* 10 (round (/ dist 120.0))))
         (print div)
         (send minjerk :reset
               :position-list (list prev-av (v+ prev-av scale-av))
               :time-list (list tm))
         (send minjerk :start-interpolation)
         (send minjerk :pass-time (/ tm div))
         (dotimes (i div)
           (setq avs (append avs (list (send minjerk :pass-time (/ tm div))))))
)
```